src/pkg/packaging/deb/package.targets

```diff
@@ -4,12 +4,19 @@
 
   <UsingTask TaskName="ReplaceFileContents" AssemblyFile="$(LocalBuildToolsTaskDir)core-setup.tasks.dll"/>
 
+  <PropertyGroup>
+    <IsDebianBasedDistro Condition="$(PackageTargetRid.StartsWith('ubuntu')) or
```

Comment:
Are you sure PackageTargetRid is set by the time you get here? I thought it was DistroRid that was the one getting set (although it is a mess so I'm not clear on that).